pkg/ui/react-app/src/thanos/pages/notFound/NotFound.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import PathPrefixProps from '../../../types/PathPrefixProps';
+
+interface NotFoundProps {
+  default: boolean;
+  defaultRoute: string;
+}
+
+const NotFound: FC<NotFoundProps & PathPrefixProps> = ({ pathPrefix, defaultRoute }) => {
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        left: '50%',
+        top: '50%',
+        transform: 'translate(-50%, -50%)',
+      }}
```

Comment:
`position: absolute` must not be used unless it's actually needed, It can cause a lot of problems. Try using
```css
width: 100%;
min-height: 100vh;
display: flex;
flex-direction: column;
justify-content: center;
align-items: center;
```

Second point, we use CSS modules for styling so don't use inline CSS unless it's necessary.